scripts/apps/authoring/metadata/metadata.js

```diff
@@ -650,6 +656,8 @@ function MetaTermsDirective(metadata, $filter, $timeout) {
                 scope.tree = tree;
                 scope.activeTree = tree.null;
                 scope.combinedList = _.union(scope.list, scope.item[scope.field] ? scope.item[scope.field] : []);
+
+                scope.$applyAsync();
```

Comment:
why?

Reply:
Is not updated the set value

Reply:
that's strange, if it's in watch callback, there should be digest in progress